scope_registry.go

```diff
@@ -263,17 +263,17 @@ func (r *scopeRegistry) reportInternalMetrics() {
 	histograms.Add(rootHistograms.Load())
 
 	if r.root.reporter != nil {
-		r.root.reporter.ReportCounter(r.sanitizedCounterCardinalityName, internalTags, counters.Load())
-		r.root.reporter.ReportCounter(r.sanitizedGaugeCardinalityName, internalTags, gauges.Load())
-		r.root.reporter.ReportCounter(r.sanitizedHistogramCardinalityName, internalTags, histograms.Load())
+		r.root.reporter.ReportGauge(r.sanitizedCounterCardinalityName, internalTags, float64(counters.Load()))
```

Comment:
Make sure internalTags is consistent with the internal tags emitted from the m3 reporter side code

Reply:
done